wallets/provider-all/src/index.ts

```diff
@@ -31,15 +30,21 @@ import * as tronLink from '@rango-dev/provider-tron-link';
 import * as trustwallet from '@rango-dev/provider-trustwallet';
 import * as walletconnect2 from '@rango-dev/provider-walletconnect-2';
 import * as xdefi from '@rango-dev/provider-xdefi';
+import {
+  legacyProviderImportsToVersionsInterface,
+  type Versions,
+} from '@rango-dev/wallets-core/utils';
```

Comment:
importing using `@rango-dev/wallets-core/utils` is using ESM's `exports` feature which is what we use from now on. We only directly use `@rango-dev/wallets-core` whenever the environment doesn't support this feature. Currently, `embedded`  is the only env that doesn't support for this. In future we will migrate that one as well since it needs some major changes and is out of scope of this task.